tests/Integration/Behaviour/Features/Scenario/Product/Shop/shop_management.feature

```diff
@@ -179,45 +158,8 @@ Feature: Copy product from shop to shop.
       | en-US  | Just a nice mug |
     And product productWithBasic is not associated to shop shop3
     And product productWithBasic is not associated to shop shop4
-    # Now modify and copy the values but this time the shop is already associated so it is an update
-    When I update product "productWithBasic" with following values:
-      | name[en-US]              | photo of super mug |
-      | description[en-US]       | super mug          |
-      | description_short[en-US] | Just a super mug   |
-    Then product "productWithBasic" localized "name" for shops "shop1" should be:
-      | locale | value              |
-      | en-US  | photo of super mug |
-    And product "productWithBasic" localized "description" for shops "shop1" should be:
-      | locale | value     |
-      | en-US  | super mug |
-    And product "productWithBasic" localized "description_short" for shops "shop1" should be:
-      | locale | value            |
-      | en-US  | Just a super mug |
-    But product "productWithBasic" localized "name" for shops "shop2" should be:
-      | locale | value              |
-      | en-US  | photo of funny mug |
-    And product "productWithBasic" localized "description" for shops "shop2" should be:
-      | locale | value    |
-      | en-US  | nice mug |
-    And product "productWithBasic" localized "description_short" for shops "shop2" should be:
-      | locale | value           |
-      | en-US  | Just a nice mug |
-    # Copy values to a shop which is already associated
```

Comment:
That's the use case I mentioned which is not possible anymore Although I don't think it's that important for now and we can always improve the `SetProductShopsCommand` later by adding an optional parameter `$forceOverwriteAllShops` or something similar